gamestonk_terminal/economy/economy_controller.py

```diff
@@ -20,7 +30,52 @@
 class EconomyController:
     """Economy Controller"""
 
-    # Command choices
+    l_GROUPS = [
+        "Sector",
+        "Industry",
+        "Industry (Basic Materials)",
+        "Industry (Communication Services)",
+        "Industry (Consumer Cyclical)",
+        "Industry (Consumer Defensive)",
+        "Industry (Energy)",
+        "Industry (Financial)",
+        "Industry (Healthcare)",
+        "Industry (Industrials)",
+        "Industry (Real Estate)",
+        "Industry (Technology)",
+        "Industry (Utilities)",
+        "Country (U.S. listed stocks only)",
```

Comment:
As you said, I think stripping Industry would make it more intuitive

Reply:
Just did. Had to add an additional dictionary, but it made sense. Good call